### DISCUSSION TITLE
Column Mode

### DISCUSSION THREAD

**Discussion by RolandHughes** [notepadqq/notepadqq#993] [Q&A]
Does column mode exist in notepadqq? I've seen a few messages scattered around the Internet, but can't for the life of me find any documentation or get it to work.

**Comment by ghurandom** [comment]
Hold down 'alt' and drag with cursor, same way as np++

**Comment by RolandHughes** [comment]
Thanks,

I stumbled into that.

Roland

On 4/8/2021 4:17 AM, ghurandom wrote:
>
> Hold down 'alt' and drag with cursor, same way as np++
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/notepadqq/notepadqq/discussions/993#discussioncomment-583709>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AAOLYVH3PEX74HOP3TDWTCDTHVYDTANCNFSM4Z7KPDMQ>.
>
-- 
Roland Hughes, President
Logikal Solutions
(630)-205-1593  (cell)
http://www.theminimumyouneedtoknow.com
http://www.infiniteexposure.net
http://www.johnsmith-book.com

**Comment by RolandHughes** [comment]
If NP didn't change the default Scintilla list, pull down RedDiamond 
from SourceForge.net. One of the menu options shows you all of the 
shortcuts.

On 11/2/2023 4:48 AM, Michel Doffagne wrote:
>
> Hi,
> At first I was surprise, because I had the habit of using the column 
> selection with the keyboard on NP++ (CTRL+ALT+cursor), witch does not 
> seem to work here.
> Many other keyboard shortcut are not present.
> Is there a shortcut list somewhere ?
>
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/notepadqq/notepadqq/discussions/993#discussioncomment-7454621>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AAOLYVALJC444K4UNPQXMVLYCNT6VAVCNFSM4Z7KPDM2U5DIOJSWCZC7NNSXTOKENFZWG5LTONUW63SDN5WW2ZLOOQ5TONBVGQ3DEMI>.
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>
>
-- 
Roland Hughes, President
Logikal Solutions
(630)-205-1593  (cell)
http://www.theminimumyouneedtoknow.com
http://www.infiniteexposure.net
http://www.johnsmith-book.com


-- 
This email has been checked for viruses by Avast antivirus software.
www.avast.com
